### PR TITLE
completed react-rendering-lists

### PR DESCRIPTION
**Pull request by m-Cho1**
image

### CODE REVIEW

File: react-rendering-lists/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+const pokemonLists = pokedex.map(pokemon =>
+  <li key={pokemon.number}>{pokemon.name}</li>
+);
+
```
Comment (scbowler) [inline react-rendering-lists/src/index.jsx:15]: The list of `li`s should be wrapped in a `ul`
```JSX
const list = (
  <ul>
    {pokemonLists}
  </ul>
);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one update needed. Let me know if you have any questions.

**Comment by m-Cho1** [comment]
created ul, and added li

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**